src/osdc/Objecter.cc

```diff
@@ -3973,6 +3973,29 @@ void Objecter::delete_pool_snap(
   pool_op_submit(op);
 }
 
+void Objecter::delete_pool_snap_again(
+  int64_t pool,
+  decltype(PoolOp::onfinish)&& onfinish)
+{
+  unique_lock wl(rwlock);
+  ldout(cct, 10) << "delete_pool_snap; pool: " << pool << dendl;
```

Comment:
Can the log be moved to before the lock?

Reply:
removed